src/winston-logger.ts

```diff
@@ -1,8 +1,9 @@
 import * as winston from 'winston';
 import * as DailyRotateFile from 'winston-daily-rotate-file';
-import { LogstashTransport } from 'winston-logstash-transport';
 import { LoggerConfiguration } from './configurations/logger-configuration';
 
+import LogstashTransport = require('logstash-tcp-wins');
```

Comment:
No require imports, change to `import * as LogstashTransport  from 'logstash-tcp-wins';`

Reply:
then we will need to turn on `esModuleInterop` - since this package exporting via default es5 export in this package